### DISCUSSION TITLE
Is there anyway to verify domain?

### DISCUSSION THREAD

**Discussion by soxft** [plausible/analytics#1904] [Self-Hosted Support] [answered]
I found that in this version, anyone who adds a tracking script to any websites, simply sets the "data-domain" to my domain name in the script tag so it will be recorded in the backend. is there any way to prevent this things?
Maybe  I did‘t find some settings, I hope I can get help, thanks

**Comment by zkeq** [comment]
I reappear this locally

![1](https://user-images.githubusercontent.com/62864752/168426650-e948b5d6-1946-4469-b8dd-0bff36d72058.png)
![2](https://user-images.githubusercontent.com/62864752/168426648-db12df37-389e-4579-9f82-b28015499619.png)

**Comment by metmarkosaric** [comment, marked as answer]
there's more info on this topic in this thread: https://github.com/plausible/analytics/discussions/183

**Reply by soxft** [reply]
ok, let me look into it a bit more
